### PR TITLE
Natural Language GAPIC client library

### PR DESCRIPTION
**Pull request by dizcology**
migrate samples to natural language gapic client library

Merge only after the client library has been released and tested.

### CODE REVIEW

File: language/cloud-client/v1beta2/snippets.py [hunk 1]
```diff
@@ -175,17 +177,15 @@ def entity_sentiment_text(text):
...
-    language_client = language_service_client.LanguageServiceClient()
-    document = language_service_pb2.Document()
+    client = language_v1beta2.LanguageServiceClient()
 
-    document.gcs_content_uri = gcs_uri
-    document.type = enums.Document.Type.PLAIN_TEXT
+    document = types.Document(gcs_content_uri=gcs_uri, type='PLAIN_TEXT')
 
-    encoding = enums.EncodingType.UTF32
+    encoding = 'UTF32'
```
Comment (jonparrott) [inline language/cloud-client/v1beta2/snippets.py]: can you not use the enums here (should be `types.EncodingType.UTF32`?

**Comment by jonparrott** [comment]
This LGTM, but you need to resolve the conflicts first.

File: language/cloud-client/v1beta2/snippets.py (hunk 1 above)
Comment (dizcology) [inline language/cloud-client/v1beta2/snippets.py]: removing encoding, since it is no longer needed:  https://github.com/googleapis/googleapis/pull/408

Comment (dizcology) [inline language/cloud-client/v1beta2/snippets.py]: turns out we should still keep the encoding.

The client libraries are current designed to have all enums in the enums module, such as `google.cloud.language.enums`.

**Comment by dizcology** [comment]
client library version 0.26.1 released, and tested against the updated samples.

**Pull request merged**